src/main/io/vtx_string.c

```diff
@@ -24,16 +24,18 @@
 
 #include "platform.h"
 #include "build/debug.h"
+#include "common/utils.h"
 
 #if defined(VTX_COMMON)
 
-const uint16_t vtx58frequencyTable[5][8] =
+const uint16_t vtx58frequencyTable[6][8] =
 {
     { 5865, 5845, 5825, 5805, 5785, 5765, 5745, 5725 }, // Boscam A
     { 5733, 5752, 5771, 5790, 5809, 5828, 5847, 5866 }, // Boscam B
     { 5705, 5685, 5665, 5645, 5885, 5905, 5925, 5945 }, // Boscam E
     { 5740, 5760, 5780, 5800, 5820, 5840, 5860, 5880 }, // FatShark
     { 5658, 5695, 5732, 5769, 5806, 5843, 5880, 5917 }, // RaceBand
+    { 5362, 5399, 5436, 5473, 5510, 5547, 5584, 5621 }, // LowRace
 };
 
 const char * const vtx58BandNames[] = {
```

Comment:
Use `VTX_SA_BAND_COUNT` as array size - it will generate warning if number of entries does not match

Reply:
Sorry, didn't check it that deep.

It would be good idea to add `STATIC_ASSERT` into code that uses this table to check possible size mismatch.